packages/cypher-builder/src/clauses/Unwind.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Cypher from "../Cypher";
+
+describe("CypherBuilder UNWIND", () => {
+    test("UNWIND Movies", () => {
+        const matrix = new Cypher.Map({ title: new Cypher.Literal("Matrix") });
+        const matrix2 = new Cypher.Map({ title: new Cypher.Literal("Matrix 2") });
+        const moviesList = new Cypher.List([matrix, matrix2]);
```

Comment:
Interesting, I don't thought about it before honestly,
I would say that in this case both  solutions works, apart from the API be more intuitive, consider also the case:
```
new Cypher.List(null) // should it raise an Error or be coerced as NULL or be coerced as [] ?
new Cypher.Literal(null) // I think we all agree it should coerced as "NULL"
```
Another consideration is that using the List API, you could avoid to use an extra array, for instance:
```
const myArray = [1, 2, 3];
if(addExtraNumber) {
    myArray.push(4);
}
return new Cypher.Literal(myArray);
```
but using the List API it could be:
```
const myList = new Cypher.List([1, 2, 3]);
if(addExtraNumber) {
    myList.push(4);
}
return myList; // then it can be mutated from the caller as well
```